dev/benchmarks/macrobenchmarks/lib/src/web/recorder.dart

```diff
@@ -728,6 +728,7 @@ class Profile {
       final Timeseries timeseries = scoreData[key];
       final TimeseriesStats stats = timeseries.computeStats();
       json['$key.average'] = stats.average;
+      json['$key.outlierAverage'] = stats.outlierAverage;
```

Comment:
Just wanted to remind you that this won't be visible in the dashboard because you didn't add `$key.outlierAverage` to the `scoreKeys` list. Is that intended?

Reply:
Oops! Done.